app/audit/audit-styles.css

```
@@ -241,1946 +229,923 @@
+.auth-dialog-backdrop {
+  position: fixed; inset: 0; z-index: 10000;
+  display: grid; place-items: center;
+  padding: 32px 16px;
+  background:
+    radial-gradient(ellipse 1000px 700px at 30% 20%, rgba(228, 88, 124,0.08) 0%, transparent 60%),
+    rgba(8,8,10,0.78);
+  backdrop-filter: blur(6px);
+  -webkit-backdrop-filter: blur(6px);
+  animation: authFadeIn 140ms ease-out;
+}
+@keyframes authFadeIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "authFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)